src/btree/bt_split.c

```diff
@@ -1665,6 +1665,9 @@ __wt_multi_to_ref(WT_SESSION_IMPL *session,
 	 * overrides WT_REF_DISK.
 	 */
 	if (multi->las_pageid != 0) {
+		WT_ASSERT(session, multi->disk_image == NULL);
+		WT_ASSERT(session, multi->addr.addr != NULL);
```

Comment:
@michaelcahill, the assert of `addr.addr` backs out the change in WT-3663 (#3739)?

I definitely saw pages that were only lookaside records, without an image or backing block.

Reply:
Thanks @keithbostic, that was just sanity checking, I didn't intend for it to be merged.  Interestingly, I haven't seen it fire yet.  I've backed it out regardless.